src/DynamoApplications/StartupUtils.cs

```diff
@@ -197,13 +206,17 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
         /// which if we are running on mac os or *nix will use different logic then SandboxLookup
         /// </summary>
         /// <param name="hostName">Dynamo variation identified by host.</param>
+        /// <param name="parentId"></param>
+        /// <param name="sessionId"></param>
         /// <returns></returns>
-        private static IUpdateManager InitializeUpdateManager(string hostName = "")
+        private static IUpdateManager InitializeUpdateManager(string hostName = "", String parentId = "", String sessionId = "")
         {
```

Comment:
yeah, weird place to put them, UpdateManager
Can you add a separate jira task to refactor this (move them out) ?
Or maybe we can do it as part of this PR ?

Reply:
Will give that a try